arc_solvers/training_config/qa/multi_choice/reader_qa_multi_choice_max_att_ARC_Chellenge_full.json

```diff
@@ -0,0 +1,86 @@
+{
+  "dataset_reader": {
+    "type": "arc-multi-choice-json",
+    "token_indexers": {
+        "tokens": {
+            "type": "single_id",
+            "lowercase_tokens": true
+        }
+    },
+    "tokenizers": {
+        "default": {
+            "start_tokens": ["@start@"],
+            "end_tokens": ["@end@"]
+        },
+    },
+  },
+  "train_data_path": "../data/allenai/ARC-V1-Feb2018/ARC-Challenge/ARC-Challenge-Train.jsonl",
+  "validation_data_path": "../data/allenai/ARC-V1-Feb2018/ARC-Challenge/ARC-Challenge-Dev.jsonl",
```

Comment:
To make these directly run-able, you could put this data in S3 and use the `cached_path` function in allennlp which would automatically download data not present locally to a cache.

Reply:
We already download the data for the other solvers so it would be inconsistent if we change it only here. Also, the original dataset release contains many files and we do not want to add an S3 file for each file of this dataset distribution. Can we have a way for "deep linking" to a specific file in an archive in S3?

Reply:
Sure - I didn't realise that. No problem 👍